src/main/webapp/app/view/project/screenplay/filter/project-screenplay-filter.component.html

```diff
@@ -13,6 +13,32 @@
                     </ng-template>
                 </button>
             </li>
+
+            <li class="nav-item w-100">
```

Comment:
Besser wäre die `li` eine Bootstrap Grid Klasse zu geben, statt w-100. Würde allen `li` die Klasse `col-12 col-sm-6 col-md-3` geben. Somit sind auf Handys die Filter die ganze Breite, bei kleinen Tablets 2 auf der Breite, ansonsten 4.
https://getbootstrap.com/docs/4.0/layout/grid/

Reply:
sounds good to me. I'll try it.